openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -1086,7 +1086,7 @@
     }
 
     .action--save {
-      @extend %action-button
+      @extend %action-button !optional
     }
 
     .feedback {
```

Comment:
I've poked around - no files are obviously AI grading exclusive, and I can't see any obvious removals in the existing files. Judging by that and the fact that very little HTML needed to be removed, I think we're safe assuming that this feature never had a lot of dedicated front-end customization done.